pkg/kubelet/prober/worker.go

```diff
@@ -62,6 +63,11 @@ type worker struct {
 	resultRun int
 }
 
+const (
+	// jitter factor for worker.doProbe sync loop
+	workerProbeJitterFactor = 1.0
```

Comment:
If kubelet restarted the probes could be started in rapid succession. What if we just sleep for a random time up to probe period before starting the probe? I think that would have the same net effect without being user-visible.